src/Pages/ProcessPage.js

```diff
@@ -0,0 +1,320 @@
+import {
+  FormControl,
+  FormGroup,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  Snackbar,
+  Typography,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { Alert } from '@material-ui/lab';
+import ProcessCard from 'components/common/ProcessCard';
+import { getCropCyclesByField } from 'dataclients/CropCyclesClient';
+import { getFieldsByLocation } from 'dataclients/FieldsClient';
+import { getLocations } from 'dataclients/LocationsClient';
+import { getProcessStatuses } from 'dataclients/ProcessesClient';
+import React, { useEffect, useState } from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  gridItem: {
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  formControl: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    minWidth: 300,
+    maxWidth: 300,
+  },
+  submitButton: {
+    margin: theme.spacing(3),
+    width: 'fit-content',
+    minWidth: '150px',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  menuItem: {
+    maxWidth: 300,
+    whiteSpace: 'normal',
+  },
+  helpText: {
+    fontSize: theme.typography.pxToRem(14),
+    color: theme.palette.text.secondary,
+  },
+}));
+
+const ProcessPage = () => {
+  const classes = useStyles();
+  const [locationCode, setLocationCode] = useState('');
+  const [locations, setLocations] = useState([]);
+  const [fieldId, setFieldId] = useState('');
+  const [fields, setFields] = useState([]);
+  const [cropCycleId, setCropCycleId] = useState('');
+  const [cropCycles, setCropCycles] = useState([]);
+  const [processStatuses, setPprocessStatuses] = useState([]);
+  const [alertMessage, setAlertMessage] = useState('');
+  const [alertStatus, setAlertStatus] = useState(false);
+  const [alertSeverity, setAlertSeverity] = useState('');
+
+  const handleAlertClose = (_event, reason) => {
```

Comment:
functions related to alert are there in every page, should we make it a separate component?